src/questions/dtos/question.dto.ts

```diff
@@ -0,0 +1,34 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsNumber } from 'class-validator';
+
+export class CreateQuestionDto {
+  @ApiProperty()
+  @IsNotEmpty()
+  stack_id: string;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  @IsNumber()
+  max_submit_time: number;
```

Comment:
parameter not use snake case, just use camelCase as usually but when user input, use kebab in json